mobile/app.json

```diff
@@ -28,15 +28,15 @@
           }
         }
       },
-      "buildNumber": "9"
+      "buildNumber": "12"
```

Comment:
Regarding this field, I don't know yet if we have to keep it updated or not (I think it should be like "version", so no updated).

Reply:
https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md